### PR TITLE
Fixes to invenio-deposit

### PR DESCRIPTION
**Pull request by fenekku**

### CODE REVIEW

File: invenio_deposit/static/js/invenio_deposit/app.js [hunk 1]
```diff
@@ -24,12 +24,10 @@
...
   // Bootstrap it!
   angular.element(document).ready(function() {
+
     angular.bootstrap(
       document.getElementById("invenio-records"), [
-        'invenioRecords', 'schemaForm', 'mgcrea.ngStrap',
-        'mgcrea.ngStrap.modal', 'pascalprecht.translate', 'ui.select',
-        'mgcrea.ngStrap.select', 'invenioFiles'
-      ]
+      'invenioRecords', 'schemaForm', 'mgcrea.ngStrap', 'mgcrea.ngStrap.modal', 'pascalprecht.translate', 'ui.sortable', 'ui.select', 'mgcrea.ngStrap.select', 'mgcrea.ngStrap.datepicker', 'mgcrea.ngStrap.helpers.dateParser', 'mgcrea.ngStrap.tooltip', 'invenioFiles'      ]
```
Comment (phebal) [inline invenio_deposit/static/js/invenio_deposit/app.js]: I don't want to nitpick but why is this really long line better then the above?

File: .gitignore [hunk 2]
```diff
@@ -66,3 +66,6 @@ examples/data/upload/*
 
 # temporary db
 tests/*.db
+
+# Editors
+*.sublime-*
```
Comment (phebal) [inline .gitignore]: Missing newline

File: invenio_deposit/mappings/v6/deposits/deposit-v1.0.0.json [hunk 3]
```diff
@@ -3,15 +3,14 @@
...
-            "index": "not_analyzed"
+            "type": "text",
+            "index": false
         },
         "_deposit": {
           "type": "object",
           "properties": {
             "id": {
-              "type": "string",
-              "index": "not_analyzed"
```
Comment (phebal) [inline invenio_deposit/mappings/v6/deposits/deposit-v1.0.0.json:14]: do we want this in the index?

**Review by phebal — APPROVED** [review]
Looks good

File: invenio_deposit/mappings/v6/deposits/deposit-v1.0.0.json (hunk 3 above)
Comment (fenekku) [reply invenio_deposit/mappings/v6/deposits/deposit-v1.0.0.json:14]: Good question. Invenio-deposit had it, so I kept it. Those changes are just to make it compatible with ElasticSearch 6.

File: .gitignore (hunk 2 above)
Comment (fenekku) [reply .gitignore]: Fixed.

File: invenio_deposit/static/js/invenio_deposit/app.js (hunk 1 above)
Comment (fenekku) [reply invenio_deposit/static/js/invenio_deposit/app.js]: It is not! My editor was wrapping those lines. Fixed.

**Comment by phebal** [comment]
Thanks for the fixes!

**Pull request merged**